src/main/java/org/example/student/Student.java

```diff
@@ -0,0 +1,102 @@
+package org.example.student;
+
+import org.example.course.Course;
+import org.example.department.Department;
+
+import java.util.Set;
+
+public class Student {
+
+    public enum Gender {
+        MALE,
+        FEMALE
+    }
+
+    private String name;
+    private String surname;
+    private String studentId;
+    private Department department;
+    private Set<Course> courses;
+    private int age;
+    private Gender gender;
+
+    public Student() {
+    }
+
+    public Student(String name, String surname, String studentId, int age, Gender gender) {
```

Comment:
shouldn't it also have a Department?
And for the Courses - > you could also write here:
courses = new HashSet<>(); 
so that the courses object is not null, but a empty collection